docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  calypso:
+    image: node:6.10.0
+    extra_hosts:
+      - 'calypso.localhost:0.0.0.0'
+    environment:
+      CALYPSO_ENV: 'development'
+      NODE_ENV: 'development'
+      HOST: 'calypso.localhost'
+      PORT: 3000
+    ports:
+      - '3000:3000'
```

Comment:
I agree that it should belong to firewalls, but docker plays with iptables to open up ports and this has caused enough problems previously, see: https://github.com/docker/docker/issues/22054